scripts/Start-EditorServices.ps1

```diff
@@ -71,8 +71,23 @@ param(
     $ConfirmInstall
 )
 
-function ExitWithError($errorString) {
+$minPortNumber = 10000
+$maxPortNumber = 30000
+
+if ($LogLevel -ne "Normal") {
```

Comment:
There are "lesser" log levels than Normal (though honestly they're kinda pointless), so maybe you should check for Verbose and Diagnostic specifically?  Or maybe just Diagnostic?  Can definitely change this in a separate PR so that the release isn't held up.

Reply:
Yeah, I think "Diagnostic" is the right level to enable the logging since we'd only use that level in cases where things have _really_ gone sideways :)

Reply:
I'll change this tonight.  BTW how do we handle the fact that this file had an Authenticode signature which I had to remove because my changes invalidated it?

Reply:
@tylerl0706 needs to sign the script again when he ships the update.  In the past I wasn't checking in the authenticode part, but as time went on it seemed easier to just check it in since the script doesn't change that often between releases.

Reply:
On the VSCode side, `Diagnostics` isn't one of the logging options although I suppose we could add it:

![image](https://user-images.githubusercontent.com/5177512/36460697-cb963682-1676-11e8-828c-0f3dd3a49741.png)

Thoughts?  `Diagnostics`  is available on the PSES side.